src/Layer/include/SkiaImpl/VSkFontMgr.h

```diff
@@ -230,7 +230,7 @@ class VGGFontLoader : public SkFontMgrVGG::SystemFontLoader
     {
       SkFontStyleSet_VGG* family = new SkFontStyleSet_VGG(SkString());
       families->push_back().reset(family);
-      family->appendTypeface(sk_make_sp<SkTypeface_VGG_Empty>());
+      family->appendTypeface(sk_ref_sp(new SkTypeface_VGG_Empty()));
```

Comment:
Could you please give more reason about switching from `sk_make_sp` to `sk_ref_sp(new ...`

Reply:
```sk_make_sp``` does not increase the reference counter automatically. The allocated object is not mananged by that form. It could lead double free or memory leak.  Another feasible way is
`auto obj = sk_make_sp<T>();
if(obj){
obj->ref();
}
`
```sk_ref_sp``` is just a short form.

Reply:
It turns out that `sk_make_sp` is just a stupid wrapper (see 'core/SkRefCnt.h:370'). We should pay more attention if we want to use skia's internal functions in the future.